### DISCUSSION TITLE
Add a delay timer to the Element Picker (Manifest v2 version)

### DISCUSSION THREAD

**Discussion by dftf-stu** [uBlockOrigin/uBlock-issues#3460] [Ideas] [answered]
Would it be possible to add a delay/countdown timer to the Element Picker feature, please?

Currently, the moment you click the Element Picker icon (the eye-drop) the entire webpage
gets dimmed, and you can hover-over a part of the page.

However... on some websites, they have "hover-over" elements which I sometimes trigger
accidentally, and find they take too-long to then disappear.  I'd like to block these, but it
seems there is no-current way to select them, as the moment I click on the uBO extension,
they disappear.

What would be great is a countdown mode, similar to what the "Snipping Tool" features,
where the webpage would dim after say 10 seconds, giving me time to bring the hover-over
element on-screen, so I can then select it.

![SnS_CDTimer](https://github.com/user-attachments/assets/0ddbe71b-951a-4896-9322-b15ccd55a7aa)

Thanks

_Applies to:_ uBlock Origin browser-extension (Manifest v2 version)
_Not applicable to:_ uBlock Origin Lite browser-extension (Manifest v3 version)

**Comment by gorhill** [comment]
> on some websites

Provide real, actual examples of cases so that we can for ourselves the issue you are reporting:
- Provide URL of webpages where the issue can be reproduced
- Explain how to reproduce the issue on those specific webpages

**Comment by dftf-stu** [comment]
@gorhill : one easy example: Reddit

Go to [https://www.reddit.com/r/LinusTechTips/comments/1gv497w/phew_that_was_close/](https://www.reddit.com/r/LinusTechTips/comments/1gv497w/phew_that_was_close) for example,
and hover-over any of the user's profile names, and a "contact card" appears.

Sometimes these disappear within a second or two of me moving the mouse away, but other
times they can literally get stuck for around 10 seconds, at-least.  Even hovering-over other users
doesn't always make the older one disappear.

![RedditLTT](https://github.com/user-attachments/assets/f347f027-4f6b-4020-9a2c-825541573c16)

The part in the red box is the element I'd like to block -- but I cannot see how to do this, as it will
always disappear before I can select it.

Thanks

**Reply by uBlock-user** [reply, marked as answer]
Duplicate of https://github.com/gorhill/uBlock/issues/2000

**Reply by gorhill** [reply]
It's not something uBO can control, the site executes JS for that overlay to show.

The solution is to use your browser's DOM inspector, and in this very case it requires to understand HTML/CSS, including shadow DOM. I could make the overlay to stick. In such cases, best to seek help on support forums, some people will know how to accomplish what you want.

So if what you want is to remove these overlays, try:

    www.reddit.com##[faceplate-popper-content]

**Reply by uBlock-user** [reply]
@gorhill So https://github.com/gorhill/uBlock/issues/2000 cannot be addressed ?

**Reply by gorhill** [reply]
@uBlock-user Not a duplicate, the case here uses JS to change the DOM, while the issue is strictly for CSS-driven overlays, like those using `:hover`.

**Reply by uBlock-user** [reply]
`:hover` is involved as per "rules" column in inspector pane in devtools and can be seen in action when I hover over the username and card appears, devtools shows `:hover` active in yellow color

**Comment by dftf-stu** [comment]
@gorhill : apologies for the duplicate.

When I went to the **uBlockOrigin** repo and searched for open tickets there, I didn't find anything matching this issue.

It wouldn't have occurred to me to also look in the **uBlock** one, as I assumed that would be for either a different product,
or for like the actual core-code of the product.  Sadly, this often gets confusing for end-users like me to know _where_ to look
for projects that have multiple repos all related to what to us seems like one end-user product!

**Comment by dftf-stu** [comment]
Advised duplicate of: https://github.com/gorhill/uBlock/issues/2000
